test/unit.js

```diff
@@ -1036,6 +1034,252 @@ describe('Populator with specified unique tracked entity attribute ID with multi
   });
 });
 
+describe('Populator with specified unique data element ID', function() {
+  describe('#typeCache', function() {
+    it('should set specified uniqueDataElementID to the cache', function (next) {
+      var populatorOptions =
+      {
+        uniqueDataElementID: 'data element id'
+      };
+      var populator = new Populator(populatorOptions);
+      expect(populator._cache.uniqueEventDataElementID).to.equal('data element id');
+      expect(populator._options.duplicateThreshold).to.equal(-1)
+      next();
+    })
+
+    it('should not set uniqueTrackedEntityAttributeID to cache if option not passed in', function (next) {
```

Comment:
The title of the test does not match what is being tested.
`uniqueTrackedEntityAttributeID` -> `uniqueEventDataElementID`